db/transforms/base.py

```diff
@@ -257,32 +257,40 @@ def map_of_output_alias_to_input_alias(self):
     def apply_to_relation(self, relation):
 
         def _get_grouping_column(col_spec):
-            preproc = col_spec.get('preproc')
-            out_alias = col_spec.get('output_alias')
-            in_alias = col_spec['input_alias']
-
-            expr = relation.columns[in_alias]
-
-            if preproc is not None:
-                expr = get_db_function_subclass_by_id(preproc).to_sa_expression(expr)
-            if out_alias is not None:
-                expr = expr.label(out_alias)
-
-            return expr
+            preproc_db_function_subclass_id = col_spec.get('preproc')
+            input_alias = col_spec['input_alias']
+            output_alias = col_spec['output_alias']
```

Comment:
Why has `output_alias` become required? It's not needed for grouping columns in most cases.

Reply:
It makes things simpler. I ran into situations where I had to choose whether to install checks and handling in case `output_alias` isn't provided or to hope that frontend doesn't stop providing it (admittedly, I chose to hope, iirc). If we constrain the spec, rest of code becomes simpler.